packages/scoped-elements/src/registerElement.js

```diff
@@ -1,55 +1,104 @@
 import { createUniqueTag } from './createUniqueTag.js';
 
 /**
- * Global cache for tag names
+ * @typedef {function(): Promise<typeof HTMLElement>} Lambda
```

Comment:
I suggest calling this LazyElement or LazyElementFn

Reply:
+1 to LazyElementFn